dev/devicelab/manifest.yaml

```diff
@@ -411,14 +411,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
 
-  # flutter_gallery_instrumentation_test:
-  #   description: >
-  #     Same as flutter_gallery__transition_perf but uses Android instrumentation
-  #     framework, and therefore does not require a host computer to run. This
-  #     test can run on off-the-shelf infrastructures, such as Firebase Test Lab.
-  #   stage: devicelab
-  #   required_agent_capabilities: ["linux/android"]
-  #   flaky: true
+  named_isolates_test:
```

Comment:
This diff looks strange.

Maybe remove `flutter_gallery_instrumentation_test` in a separate PR to make it cleaner?

Reply:
Somehow during rebase it goes in. It's removed now.